app/uk/gov/hmrc/tariffclassificationfrontend/connector/BindingTariffClassificationConnector.scala

```diff
@@ -51,21 +51,27 @@ class BindingTariffClassificationConnector @Inject()(appConfig: AppConfig, clien
     client.GET[Option[Case]](url)
   }
 
+  private def buildQueryUrl(statuses: String, queueId: String = "", assigneeId: String,
+                            paginationPage: Int, paginationSize: Int): String = {
+    val sortBy = "application.type,application.status,days-elapsed"
+    s"application_type=BTI&application_type=LIABILITY_ORDER&queue_id=$queueId&assignee_id=$assigneeId&status=$statuses&sort_by=$sortBy&sort_direction=desc&page=$paginationPage&page_size=$paginationSize"
+  }
```

Comment:
Minor, but does `application_type=BTI,LIABILITY_ORDER&queue_id....` not work?

Reply:
I'll give it a try. Was hoping to tidy that method up a little bit anyway

Reply:
Tidied up.